### DISCUSSION TITLE
How to change Five Server status bar foreground color in VS Code?

### DISCUSSION THREAD

**Discussion by mariuszciaston** [yandeu/five-server#87] [Help / Questions] [answered]
After i click "Go live" in VS Code lower right corner it starts extension and turns into http://127.0.0.1:5555/ with orange text color.

How do i change it to other color?

I tried changing/adding multiple items in settings.json but without success. I am able to change anything except this.

Also looked at https://code.visualstudio.com/api/references/theme-color but can't find solution.

Maybe this needs to be changed on extension level?

Any ideas? This is last step in my perfect light/dark theme customization i need. Thanks in advance.

**Comment by yandeu** [comment, marked as answer]
This has to be a setting in the extension. Unfortunately I do not expose this setting. But I can add it in the next release.

The code for this, is here: https://github.com/yandeu/five-server-vscode/blob/main/src/helpers.ts#L10.

**Reply by mariuszciaston** [reply]
Thanks for response, this would be great! I am waiting for it.
